cmd/kubeapps-apis/docs/kubeapps-apis.swagger.json

```diff
@@ -0,0 +1,318 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "kubeappsapis/core/packages/v1alpha1/packages.proto",
```

Comment:
Looks like the info defines this as core.packages.v1alpha1 only, whereas its describing others too (like `core.plugins.v1alpha1` and `plugins.fluxv2.packages.v1alpha1` etc.)?